### ISSUE TITLE
[bug] Extra Trash Around Model

### ISSUE THREAD

**Issue by archaeogiant** [alicevision/Meshroom#2027]
Meshroom-2012.1.0 when meshing is complete there is a bunch of trash around the model, I noticed this with some of the model I processed with Meshroom-2021.1.0 too.

This is after SFM has processed.
![SFM](https://github.com/alicevision/Meshroom/assets/134665603/0a94aa92-da42-48a9-9768-85c4cf748f3a)

This after meshing has processed
![Trash 2](https://github.com/alicevision/Meshroom/assets/134665603/3d0c260b-1ac1-46f1-8aa4-5888f01c136b)
![Trash 1](https://github.com/alicevision/Meshroom/assets/134665603/d0062b5e-eb94-4caf-9c65-efcacffa7e62)

What is going on here and how do I go about correcting this in the future?

**Comment by fabiencastan** [comment]
That's weird. Could you share your dataset to help us fix that?

**Comment by archaeogiant** [comment]
Yeah, I will be more than happy to share the dataset. It wont be until Friday though.

**Comment by patodichayan** [comment]
@archaeogiant  are you using a custom bbox? 

This is a bug for custom bboxes from what I have seen & used. 
https://github.com/alicevision/Meshroom/issues/1573

**Comment by archaeogiant** [comment]
Nope, no custom bbox used.
